src/output.ts

```diff
@@ -0,0 +1,59 @@
+import * as Serverless from "serverless";
+
+const yellowFont = "\x1b[33m";
+const underlineFont = "\x1b[4m";
+const endFont = "\x1b[0m";
+const outputPrefix = "DatadogMonitor";
+
+/**
+ * Builds the CloudFormation Outputs containing the alphanumeric key, description,
+ * and value (URL) to the function in Datadog
+ */
+export async function addOutputLinks(serverless: Serverless, site: string) {
+  const awsAccount = await serverless.getProvider("aws").getAccountId();
+  const region = serverless.service.provider.region;
+  const outputs = serverless.service.provider.compiledCloudFormationTemplate.Outputs;
+  if (outputs === undefined) {
+    return;
+  }
+
+  serverless.service.getAllFunctions().forEach((functionKey) => {
+    const functionName = serverless.service.getFunction(functionKey).name;
+    const key = `${outputPrefix}${functionKey}`.replace(/[^a-z0-9]/gi, "");
+    outputs[key] = {
+      Description: `See ${functionKey} in Datadog`,
+      Value: `https://app.${site}/functions/${functionName}:${region}:${awsAccount}:aws?source=sls-plugin`,
+    };
+  });
+}
+
+export async function printOutputs(serverless: Serverless) {
+  const stackName = `${serverless.service.getServiceName()}-${serverless.getProvider("aws").getStage()}`;
```

Comment:
Does serverless gives you a builtin method to get the CFN stackName, asking because I wonder whether the stack name **always** follow this pattern.

Reply:
There is one that I missed, that I'll get this updated to use that. That will also take into account if you manually specify the `provider.stackName`. 

https://github.com/serverless/serverless/blob/cbc5e3c4e199a08650b02def8d1f9e91a13e7844/lib/plugins/aws/lib/naming.js#L47-L54